### ISSUE TITLE
Is dreact alive?

### ISSUE THREAD

**Issue by atatarenko** [datarockets/dreact#43]
The future of dreact is not clear and it brings unwanted challenges.

Even positive contributions like [this one](https://github.com/datarockets/dreact/pull/42) faced reasonable blockage because of the unclear state of things. On one hand, this repo is not archived and it gives an impression that further usages and contributions are welcomed. On the other hand, it is a false impression because the true fact is that we would need to completely re-write the tool.

Giving all that, I propose to officially state that dreact is deprecated and archive the repo so that we can break this state of uncertainty and focus on what's more important.

**Comment by ezhlobo** [comment]
@atatarenko I agree with that, dreact is outdated way too much to continue supporting it. Feel free to archive or adjust description and other content or let me know if you want me doing that.

JFYI, there are different branches with attempts to restart it, however those require more thought process than coding. So i would not erase the repo.

**Comment by atatarenko** [comment]
@ezhlobo got it, thanks! Regarding archiving, I'd kindly ask you to do it, just to make sure it's done properly.
